shop-be/product-service/src/libs/db.ts

```diff
@@ -0,0 +1,41 @@
+import { Client, ClientConfig } from 'pg';
```

Comment:
Working solution, but please also take a look to sequalize as ORM - it's quite widespread

Reply:
will do it later) also I heard about knex and bookshelf - have you used them?

Reply:
Unfortunately no - only met sequalize on production several times